BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -72,6 +72,33 @@ + (BOXContentClient *)defaultClient
     return defaultInstance;
 }
 
++ (void)refreshDefaultClientFromKeychain
+{
+    [[[self class] SDKClients] removeAllObjects];
+    
+    NSArray *usersFromKeychain = [BOXAbstractSession usersInKeychain];
+    BOXContentClient *oldDefaultInstance = defaultInstance;
+    
+    if (usersFromKeychain.count == 0) {
+        defaultInstance = [[self alloc] init];
+    }
+    else if (usersFromKeychain.count == 1)
+    {
+        BOXUserMini *storedUser = [usersFromKeychain firstObject];
+        defaultInstance = [[self alloc] initWithBOXUser:storedUser];
+        [[[self class] SDKClients] setObject:defaultInstance forKey:storedUser.modelID];
+    }
+    else {
+        [NSException raise:@"You cannot use 'defaultClient' if multiple users have established a session."
+                    format:@"Specify a user through clientForUser:"];
+    }
+    
+    if (oldDefaultInstance.accessTokenDelegate) {
+        defaultInstance.accessTokenDelegate = oldDefaultInstance.accessTokenDelegate;
+    }
+    oldDefaultInstance.session = defaultInstance.session;
+}
```

Comment:
should we update logoutAll to also clear out in-memory user objects of each content client?